docs/_pages/releases.md

```diff
@@ -31,6 +31,7 @@ sidebar:
     * `BeGreaterOrEqualTo`: Use `BeGreaterThanOrEqualTo`
     * `BeLessOrEqualTo`: Use `BeLessThanOrEqualTo`
 * Removed the `DefaultValueFormatter.SpacesPerIndentionLevel` property which was added during the development of v6, but wasn't removed before the release of v6 - [#2281](https://github.com/fluentassertions/fluentassertions/pull/2281)
+* Dropped support for .NET Core - [#2302](https://github.com/fluentassertions/fluentassertions/pull/2302)
```

Comment:
Let's be more explicit here about what has changed.
I know several who think of ".NET Core" as everything newer ".NET Framework".
FA will still run on netcoreapp20 and above, there are just a few APIs that are no longer available.

Or better, we create a migration guide for v7 and link to that from here.